addons/web/static/src/core/install_prompt/install_prompt.scss

```diff
@@ -0,0 +1,30 @@
+.o_install_prompt {
+    border-radius: 20px;
```

Comment:
I've read your explanation about the `border-radius` value but, as it's an "unconventional" one, could you add the small comment about it ?